devops/automation/system-dependencies.sh

```diff
@@ -5,7 +5,8 @@
 
 set -o pipefail
 
-cd "$(dirname "$0")/.."
+cd "$(dirname "$0")/../.."
+echo "This is where it is cd $(dirname "$0")/../.."
```

Comment:
What is this?

Reply:
Oh man good call, I forgot to remove that echo 😮